### PR TITLE
Organisations

### PR DESCRIPTION
**Pull request by KilnOfTheSecondFlame**
Please review the language and syntax. Pull if approved, reject otherwise

### CODE REVIEW

File: Thesis.tex [hunk 1]
```diff
@@ -64,9 +64,47 @@ \section{Patents}
 \section{Trademark}
 \label{sec:Trademark}
 
-\section{Organisations}
+\section{Notable Organisations}
 \label{sec:Orgas}
 
+\subsection{BSA}
+\label{ssec:BSA}
+The Software Alliance (changed its name from \underline{B}usiness \underline{S}oftware \underline{A}lliance) is an advocate group representing the interests of the software industry in countries around the world. It was established in 1988 and has its headquarters in Washington DC. Notable members include, amongst others, Adobe, Apple, Autodesk, IBM, Microsoft, Oracle, and Siemens. The current CEO, Victoria Espinel, studied to become a telecommunications lawyer before being appointed as advisor on intellectual property issues of President Obama in 2009 and subsequent appointment to her current position inside the BSA. \parencite{Rogers2017}
```
Comment (NathDrac) [inline Thesis.tex]: Isn't it Advisor to (opposed to advisor of?) Not sure,though

Comment (NathDrac) [inline Thesis.tex]: before being appointed [] and subsequent appointment ... doesn't add up
... and subsequently being appointed ...
oder
... and her subsequent appointment ...

File: Thesis.tex [hunk 2]
```diff
@@ -64,9 +64,47 @@ \section{Patents}
...
+\noindent The BSA's mission is based on three overriding policies:
+\begin{itemize}
+	\item Safeguarding intellectual property rights and protections
+	\item Opening global markets to digital trade
+	\item Enabling the growth of cloud 
+\end{itemize}
+\parencite{BSA2017}
+
+
+To reach this stated goal the Software Alliance believes that patents, copyrights, trademarks, and other IP rights should be comprehensive and enforceable. Trading partners, whether on a global or local scale, should not use their legislation to favour local industries or discourage international competition. And lastly, companies should not abuse their intellectual property to create unfair competitive advantage.
```
Comment (NathDrac) [inline Thesis.tex]: stated goal,

File: Thesis.tex [hunk 3]
```diff
@@ -64,9 +64,47 @@ \section{Patents}
...
+
+The BSA concentrated its effort for pushing the cloud in pushing for specific policies advancing the cloud, forging bilateral or multilateral agreements that eases the movement of data across borders. It promotes the use of sound data protection practices to expand and fortify the public trust in digital commerce. But even these data protection policies should leave enough leeway for technological innovation and new services, especially cloud computing.
```
Comment (NathDrac) [inline Thesis.tex]: agreements that eases ersetzen durch agreemants that ease

File: Thesis.tex [hunk 4]
```diff
@@ -64,9 +64,47 @@ \section{Patents}
...
+
+The BSA garnered critic for aggressively enforcing the licence rights of its members under the guise of "audits" and providing bounties for people providing hints of unlicensed software use in companies. Due to high penalties for unlicensed software and the need for very extensive proof of ownership, many small and medium sized businesses failed to provide ample documentation and were subsequently fined. Some of these businesses even to the point of bankruptcy. These actions garnered the BSA the label to be akin to a bully and not the shining knight for IT innovation as stated in their mission statement. \parencite{Gaskin2009}
```
Comment (NathDrac) [inline Thesis.tex]: fined, some of

Comment (NathDrac) [inline Thesis.tex]: IT innovation as presented in their mission statement

File: Thesis.tex [hunk 5]
```diff
@@ -64,9 +64,47 @@ \section{Patents}
...
+
+\subsection{SIIA}
+\label{ssec:SIIA}
+Similar to the BSA the Software and Information Industry Association (SIIA) is a trade-group advocating and defending the rights and interests of big software companies. Notable members include, amongst many others, Apple, Facebook, Goldman Sachs and even Credit Suisse. The SIIA has the stated goal of promoting, protecting and informing the software and digital industry. Also similar to the BSA it has an anti-piracy reward program offering up to 1000000\$. \parencite{SIIAreward}
```
Comment (NathDrac) [inline Thesis.tex]: to the BSA, the Software

Comment (NathDrac) [inline Thesis.tex]: Also similar to the BSA, it has...

Comment (NathDrac) [inline Thesis.tex]: 1'000'000

File: Thesis.tex [hunk 6]
```diff
@@ -64,9 +64,47 @@ \section{Patents}
...
+
+Critics accuse the SIIA to use the complex and confusing IP laws to extort money from small- and medium sized businesses in what is essentially a shakedown. \parencite{Melymuka2006}
+
+\subsection{EFF}
+\label{sec:EFF}
+The Electronic Frontier Foundation (EFF) is a non-profit organisation and was founded in 1990 as a response to a series of raids performed by the United States Secret Service. These raids were done to track an illegally distributed manual for the emergency call system. In the course of these raids the business of a Mr. Steve Jackson was nearly driven to ruin, as they had their computer equipment confiscated, searched and tampered with. Mr. Jackson turned to civil rights groups to file a lawsuit against the Secret Service, but none of the contacted ones knew enough about the issue. Hearing about the raid and subsequent damage to Mr. Jackson's business, the three founding fathers (John Perry Barlow, John Gilmore and Mitch Kapor) announced the creation of the EFF and their intent to represent Mr. Jackson in court.
```
Comment (NathDrac) [inline Thesis.tex]: as it had their computer equipment confiscated

(subject is *the business)

File: Thesis.tex [hunk 7]
```diff
@@ -64,9 +64,47 @@ \section{Patents}
...
+The EFF continued to file lawsuits and represent individual in the upcoming decades. Their focus concern privacy rights, anti censorship, free speech and copyright issues. \parencite{EFFhistory} \parencite{Schultz}
```
Comment (NathDrac) [inline Thesis.tex]: represent individuals

File: Thesis.tex [hunk 8]
```diff
@@ -64,9 +64,47 @@ \section{Patents}
...
+
+\subsection{CC}
+\label{sec:CC}
+The Creative Commons (CC) is a non-profit organisation founded by Lawrence Lessig (a Stanford law professor) with the help of the Center for the Public Domain. Upon founding, CC released a set of licenses with which rights holder can release works to the public with various degrees of freedoms and opportunities for derivative works or permission to use this work.
```
Comment (NathDrac) [inline Thesis.tex]: a set of licenses using which right hoders

File: Thesis.tex (hunk 1 above)
Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: the act of studying was done before being appointed

File: Thesis.tex [hunk 9]
```diff
@@ -64,9 +64,47 @@ \section{Patents}
...
+In 2005 the CC launched the open science initiative with the goal to provide the chance to collaborate and contribute. In this program research data, lab notes and other research material are freely available and can be reused, redistributed and reproduced.
```
Comment (NathDrac) [inline Thesis.tex]: In this program, research data, lab ...

**Review by NathDrac — APPROVED** [review]
OK as discussed

**Pull request merged**